index.js

```diff
@@ -80,6 +80,32 @@ const parseInput = async input => {
 	return input;
 };
 
+const killChecker = (input, options) => {
+	if (typeof options.verify !== 'undefined' && options.verify === false) {
+		return;
+	}
+
+	const seconds = options.verifyTimeout || 2;
+	let checks = seconds * 10;
```

Comment:
This logic is not sound. You cannot guarantee how long things take. It's better to just continue checking until you measure using `Date()` that 2 seconds have passed.